store/tikv/coprocessor.go

```diff
@@ -582,6 +674,9 @@ func (sender *copIteratorTaskSender) run() {
 		}
 	}
 	close(sender.taskCh)
+	if sender.sendRate != nil {
```

Comment:
Is `sender.tasks != nil` better?

Reply:
Not sure. 

`sender.sendRate != nil` indicates that we need to keep an order. 
Thus `copIterator.Next` wiil be read from `sender.orderedTasksCh`. So, we need to close `sender.orderedTasksCh`. Thereby we notify `copIterator.Next` that there're no tasks any more.

Reply:
Done